Author: Marc 'HE' Brockschmidt <he@debian.org>
Description: Disable check for libgtk version - this might trigger false positives,
 our shlib-dep system is more fine-grained and correct anyway.
--- a/xs/Gtk2.xs
+++ b/xs/Gtk2.xs
@@ -125,6 +125,7 @@
 
 	/* make sure that we're running/linked against a version at least as 
 	 * new as we built against, otherwise bad things can happen. */
+	/* Not needed in Debian, we have a working shlib-dep system:
 	if ((((int)gtk_major_version) < GTK_MAJOR_VERSION)
 	    ||
 	    (gtk_major_version == GTK_MAJOR_VERSION && 
@@ -138,6 +139,7 @@
 		      " old.  We'll continue, but expect problems!\n",
 		      GTK_MAJOR_VERSION, GTK_MINOR_VERSION, GTK_MICRO_VERSION,
 		      gtk_major_version, gtk_minor_version, gtk_micro_version);
+	*/
 	}
 
 #############################################################################
